python/ray/train/tests/test_torch_trainer.py

```diff
@@ -166,15 +165,16 @@ def __init__(self, warmup_steps):
                     num_workers=num_workers,
                     use_gpu=True,
                     resources_per_worker={"GPU": num_gpus_per_worker},
-                    placement_strategy="SPREAD"
-                    # Each gpu worker will be spread onto separate nodes.
+                    placement_strategy="SPREAD",
```

Comment:
I think want you want here is more accurately described by `STRICT_SPREAD` rather than `SPREAD` that's best effort ?

Reply:
Updated!